SwiftWisdomTests/Rx/OperatorTests.swift

```diff
@@ -8,16 +8,17 @@
 
 import XCTest
 import RxSwift
+import RxCocoa
 @testable import SwiftWisdom
 
 
 class OperatorTests: XCTestCase {
 
     func testAddCompositeDisposable() {
         let compositeDisposable = CompositeDisposable()
-        let observable = Observable<String>.never()
+        let observable = Observable<String?>.never()
         XCTAssertEqual(0, compositeDisposable.count)
-        UILabel().rx_text <- observable >>> compositeDisposable
+        let _ = UILabel().rx.text <- observable >>> compositeDisposable
```

Comment:
do we have a use case for this? wouldn't this cause a lot of unused variable warnings in our code? since the purpose of >>> is to compress our code, I think that it would be better not to return disposables here.